src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -69,6 +93,8 @@ object VEP {
       else
         s)
 
+    val csqHeader = if (csq) getCSQHeaderDefinition(cmd, conf.env) else None
```

Comment:
Suggestion: Move this down below where you get the global value and type.

Reply:
I put it first so that the warning comes before all the work of VEP annotating in case users want to stop and investigate.

Reply:
ok.